Checkout

Kuchvara opened the pull request.

Kuchvara committed:
first commit

did everything except delivery logic

Kuchvara committed:
last edition

Kuchvara committed:
Merge branch 'development' into checkout

andrii-telishko commented:
@vhamolia @Kuchvara 1)When click on apply coupon button in con console I have error
                                        2)Add zoom on focused store in cart on checkout page; 
                                       3) When input.value === "" in autocomplete hide address info; 
                                         4) Check render prices in cart. Price rendering with a lot marks after point;

Kuchvara committed:
bug fixes

fixed bugs described in last comment

Kuchvara committed:
Merge branch 'checkout' of https://github.com/Kuchvara/online-market into checkout

Kuchvara committed:
last design and bug fixes

fixed bug double reviewed product
fixed bug of too long prices (after dot)
some design fixes

Kuchvara committed:
remove "console.log"
